plugin/pkg/scheduler/factory/factory.go

```diff
@@ -384,16 +384,7 @@ func (c *configFactory) onPvDelete(obj interface{}) {
 }
 
 func (c *configFactory) invalidatePredicatesForPv(pv *v1.PersistentVolume) {
-	invalidPredicates := sets.NewString("MaxPDVolumeCountPredicate")
```

Comment:
I think this needs to stay.  You could have a PVC that points to a PV, but the PV object doesn't exist.  So when the PV object gets added, we can recount.

Reply:
fixed, and also updated other comments to clarify the invalidation op reason.